tests/main.md

```diff
@@ -748,3 +748,35 @@ val b : Cstruct.t = {Cstruct.buffer = <abstr>; off = 0; len = 1}
 # Uring.exit t;;
 - : unit = ()
 ```
+
+## Timeout
+
+Timeout should return (-ETIME). This is defined in https://github.com/torvalds/linux/blob/master/include/uapi/asm-generic/errno.h#L45
+
+```ocaml
+# let t = Uring.create ~queue_depth:1 ();;
+val t : '_weak13 Uring.t = <abstr>
+
+# let ns = Int64.(mul 1L 1_000_000_000L) in
```

Comment:
It would be better to use smaller timeouts here (e.g. 10ms instead of 1s). The tests took about 400ms to run before. Adding these tests makes it 6 times slower.

Reply:
Ack.

Reply:
Done.